public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang='en'>
 <head>
+  <meta charset="UTF-8">
```

Comment:
I'm curious, does this change anything or is it just for if someone is using a wacky browser?

Reply:
https://www.w3schools.com/charsets/ref_html_utf8.asp
I specified this to be on the safe side and since I'm using special char symbols to represent my rating stars right now.  It should be the default with html5 anyways, but its just a precaution for older browsers (at least that was my logic)

Reply:
Yeah, I think it's pretty default language for an HTML template right now.